### ISSUE TITLE
New modifications

### ISSUE THREAD

**Issue by Sangi0** [multivendorx/mvx#626]
- [ ] Go to admin dashboard>>MultiVendorX>>Settings >>Commission, here in the setting "commission disbursement method", the appearance of the section needs to be changed. 
Current - http://tinyurl.com/yv6bn2tb
Updated should look like this - http://tinyurl.com/ytyanoqx

- [ ] Go to admin dashboard>>MultivendorX>>Settings>>Registration form, add afield for store name. Now while registering the vendor is filling the field of the store name and that is also set as the store name of the vendor. 
Now we want to have the store name in the store url to be set as per that instead of the email.

**Comment by Sangi0** [comment]
On the vendor store and in a product page sidebar I would like to have all of this as Vendor Info:
– Logo (already exists) Should link to the vendor store
– Name (already exists) Should link to the vendor store
– Address (already exists) Would like to show only country
– Short biography (this is the store description) Enable the admin to change the label of this for the vendor dashboard and frontend
– Rating (already exists)
– Number of sales
– Date of registration
– Social profiles (already exists)
– Link to the seller’s website

On the vendor store, tabs Products, Reviews and Policies are separate pages. Policies and Reviews are just text and do not need a full page reload. At least, load them with Ajax. But, it would be best to have just regular tabs and everything on the same page.

On the product page, the Vendor Info tab only has the Vendor Name and Store Description.
I would like to show a lot more, which I discuss in paragraph 3.
But, for a quick fix, you can update the plugin to display everything that is on the vendor store: Profile Image, Name, Rating, Country, Social, Follow and Store Description.

When adding this to the product page, please add tags and classes to these elements, so that they can be styled with CSS. Right now, the Vendor Description doesn’t have a tag or class. I assume that this should be set in the plugin’s php.
Also, the vendor store description doesn’t have the html and css markup that was added in the MCE editor. Please, fix this so that the text formatting and styling is preserved.

https://github.com/multivendorx/dc-woocommerce-multi-vendor/issues/1019
